lib/ts/controllers/s-expandable-control.ts

```diff
@@ -79,12 +79,12 @@
         };
 
 
-        get controlledCollapsible() {
+        get controlledCollapsibles() {
             const attr = this.element.getAttribute("aria-controls");
             if (!attr) {
                 throw "couldn't find controls"
             }
-            const result = document.getElementById(attr);
+            const result = attr.split(/\s+/g).map(s => document.getElementById(s)).filter((e): e is HTMLElement => !!e);
             if (!result) {
```

Comment:
```suggestion
            if (result.length == 0) {
```

Like the above, this will never fail anymore, and we could preserve the existing behavior by throwing on length.